jbpm/jbpm-flow/src/main/java/org/jbpm/marshalling/impl/ProtobufProcessMarshaller.java

```diff
@@ -183,21 +183,20 @@ public static WorkItem readWorkItem(MarshallerReaderContext context,
     public static Variable marshallVariable(MarshallerWriteContext context,
                                             String name,
                                             Object value) throws IOException {
-        JBPMMessages.Variable.Builder builder = JBPMMessages.Variable.newBuilder().setName( name );
-        if(value != null){
-            ObjectMarshallingStrategy strategy = context.objectMarshallingStrategyStore.getStrategyObject( value );
-            Integer index = context.getStrategyIndex( strategy );
-            if(strategy.getName().equals(DOCUMENT_MARSHALLING_STRATEGY)) {
-                builder.setStrategyIndex( index )
-                .setDataType(strategy.getType(value.getClass()))      
-                .setValueJson(strategy.marshalToJson(value));
-            }
-            else {
-            builder.setStrategyIndex( index )
-                   .setDataType(strategy.getType(value.getClass()))
-                   .setValue( ByteString.copyFrom( strategy.marshal( context.strategyContext.get( strategy ),
-                                                                     context,
-                                                                     value ) ) );
+        JBPMMessages.Variable.Builder builder = JBPMMessages.Variable.newBuilder().setName(name);
```

Comment:
Try to avoid pure reformatting changes if possible, otherwise you will need to insure having test cases to cover those changes.